docs/execution-providers/CoreML-ExecutionProvider.md

```diff
@@ -49,18 +60,114 @@ Ort::ThrowOnError(OrtSessionOptionsAppendExecutionProvider_CoreML(so, coreml_fla
 Ort::Session session(env, model_path, so);
 ```
 
-## Configuration Options
+## Configuration Options (NEW API)
 
 There are several run time options available for the CoreML EP.
 
 To use the CoreML EP run time options, create an unsigned integer representing the options, and set each individual option by using the bitwise OR operator.
 
+ProviderOptions can be set by passing string to the `AppendExecutionProvider` method.
```

Comment:
Given we have the code to populate the options, can we add the couple of extra lines to include the call to AppendExecutionProvider?

Reply:
Sure.